server/middlewares/validations.js

```diff
@@ -0,0 +1,77 @@
+const validateUsername = (username) => {
```

Comment:
lets have validateUsername and validateEmail in a utils file

Reply:
And add unit tests for them
And JSdoc

Reply:
Noted